Lib/compression/zstd/__init__.py

```diff
@@ -0,0 +1,293 @@
+"""Python bindings to Zstandard (zstd) compression library, the API style is
+similar to Python's bz2/lzma/zlib modules.
+"""
+
+__all__ = (
+    # From this file
+    "compressionLevel_values",
+    "get_frame_info",
+    "CParameter",
+    "DParameter",
+    "Strategy",
+    "finalize_dict",
+    "train_dict",
+    "zstd_support_multithread",
+    "compress",
+    "decompress",
+    # From _zstd
+    "ZstdCompressor",
+    "ZstdDecompressor",
+    "ZstdDict",
+    "ZstdError",
+    "get_frame_size",
+    "zstd_version",
+    "zstd_version_info",
+    # From zstd.zstdfile
+    "open",
+    "ZstdFile",
+)
```

Comment:
Consider sorting each section:

```suggestion
__all__ = (
    # compression.zstd
    "compressionLevel_values",
    "compress",
    "CParameter",
    "decompress",
    "DParameter",
    "finalize_dict",
    "get_frame_info",
    "Strategy",
    "train_dict",
    "zstd_support_multithread",

    # compression.zstd.zstdfile
    "open",
    "ZstdFile",

    # _zstd
    "get_frame_size",
    "zstd_version",
    "zstd_version_info",
    "ZstdCompressor",
    "ZstdDecompressor",
    "ZstdDict",
    "ZstdError",
)
```